Automattic/wp-calypso
# Jetpack Pricing | Release bundles pricing breakdown for all locales

In #70586, #70585, #70631 and #70983 we added some new strings and released those changes only for English locales to ensure that the UI is not broken for non-English locales. Since all of those strings have now been translated, it's time to release those changes for all locales.

#### Proposed Changes

* Remove the English locale check for pricing breakdown for Jetpack bundles on Jetpack pricing page.

#### Testing Instructions

- Do any one of these
    * Click on Jetpack Cloud live link below and wait for the redirect to complete, then goto `/pricing`
    * or boot up this PR 
        * Run `git fetch && git checkout update/release-jp-pricing-breakdown-for-all-locales`
        * Run `yarn start-jetpack-cloud`
        * Goto [http://jetpack.cloud.localhost:3000/pricing](http://jetpack.cloud.localhost:3000/pricing)
- Go to Bundles tab
- Change the language to other than English in the footer
<img width="965" alt="Screenshot 2022-12-12 at 12 19 47 PM" src="https://user-images.githubusercontent.com/18226415/206980669-0d943c28-0e3d-4d78-8e4a-48dcc570ada9.png">

- Confirm that you see the newly added Saved amount text on featured card
- Click on "More about <product name>" for Complete or Security
- Confirm that you see the pricing breakdown on the lightbox sidebar

#### Pre-merge Checklist





Related to 0-as-1203504709788035/f

<img width="1456" alt="Screenshot 2022-12-12 at 12 21 00 PM" src="https://user-images.githubusercontent.com/18226415/206981039-e37f5b01-6438-4681-9397-103d758a507d.png">
<img width="1084" alt="Screenshot 2022-12-12 at 12 25 11 PM" src="https://user-images.githubusercontent.com/18226415/206981044-05d32714-f0c6-4df0-a9a5-4e3d4d6a210f.png">


---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1203504709788035
